internals/scripts/clean.js

```diff
@@ -1,58 +1,58 @@
-require('shelljs/global');
+var shell = require('shelljs');
```

Comment:
nit: you can use `const` instead of `var` the same way as `addCheckMark` declared